client/app/scripts/superdesk-archive/views/list.html

```diff
@@ -1,11 +1,12 @@
 <div class="subnav">
 	<div class="button-stack right-stack">   
 	    <div class="navbtn dropdown strict highlights-dropdown" 
+            tooltip="{{ 'Mark item' | translate}}" tooltip-placement="left"
```

Comment:
would be good to use one time bindings for translate eg `{{ :: 'Mark item' | translate }}`